src/renderer/dx/CustomTextRenderer.cpp

```diff
@@ -318,6 +319,18 @@ try
         rect.top = rect.bottom - 1;
         break;
     }
+    case CursorType::DoubleUnderscore:
+    {
+        // Use rect for lower line.
+        rect.top = rect.bottom - 1;
+
+        // Draw upper line directly.
+        D2D1_RECT_F upperLine = rect;
+        upperLine.top -= 2;
+        upperLine.bottom -= 2;
+        d2dContext->FillRectangle(upperLine, brush.Get());

```

Comment:
This won't work with cursor colors because you're painting up here before the color is assigned below.

Reply:
Fixed, thanks.